src/common/snippets/include/snippets/lowered/port_descriptor.hpp

```diff
@@ -65,6 +65,13 @@ class PortDescriptor {
     VectorDims m_subtensor_shape{};
     /// \brief The corresponding abstract/physical register
     size_t m_reg = 0;
+
+    /// Notes: m_tensor_shape is dense shape which is controled by expression outputs.
+    ///        It means that the result of data writing of expression outputs should be read using this shape by the next expression inputs.
+    ///        Also if Port is input port of expression:
+    ///         - m_layout shows how the data should be read (by which strides) using m_tensor_shape.
+    ///        If Port is output port of expression:
+    ///         - m_layout shows how the data should be written (by which strides) to get m_tensor_shape.
```

Comment:
Please note that I'm speaking about ports connected to input/output expression, not input/ouput ports of any expression.
From the comment it follows that all input ports specify how the data is read, which could be misleading because only input ports of MemoryAccess operations specify how the data is read, other expressions are not allowed to read data.

My suggestion here is to state explicitly that only layout in Ports of MemoryAccess operations are used to reorder data. Layouts in Ports of other expressions are simply ignored for now.

Reply:
Now I see. Thank you for the explanation! Rewrote, I hope it's better now. Thanks